examples/step-44/doc/intro.dox

```diff
@@ -494,7 +497,7 @@ Thus,
 @f}
 where
 @f{align*}{
-	D_{d \mathbf{u}} R( \mathbf{\Xi}; \delta \mathbf{\Xi})
+	D_{d \mathbf{u}} R( \mathbf{\Xi}; \delta \mathbf{\Xi}) \cdot d \mathbf{u}
```

Comment:
Here and in the two places below, the formulas explain symbols used immediately above (that's the "where..." in line 498). The formula above uses `D_{du} R(...) . du` to denote a directional derivative in direction `du`, but the direction `du` was forgotten here in this line, even though the symbol `du` appears on the right hand side (three lines further down).

Same issue for the other two places below.